spec/octokit/client/repositories_spec.rb

```diff
@@ -273,6 +273,26 @@
     end
   end # .collaborators
 
+  describe ".topics", :vcr do
```

Comment:
Could we put these under the `with repository` context?

That way if we ever have to re record these cassettes another user can run them with their own data?

Reply:
Good idea 👍 

I am trying (by the way, there are two `with repository` contexts) and what [test ENVs](https://github.com/octokit/octokit.rb#running-and-writing-new-tests) do I need in order to record cassettes correctly.

It's getting closer to my sleep time. If you could show me how to move `topics` spec inside one of the context will be very helpful. I can update the other one tomorrow.

Reply:
Apologies for not getting back to this, glad you figured it out!